cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -146,10 +140,10 @@
   /**
    * Runtime program service for running and managing programs.
    */
-  private final ProgramRuntimeService runtimeService;
+  protected final ProgramRuntimeService runtimeService;
```

Comment:
should we group the protected and private members together? Makes it more readable.